xml/schema/Price/PriceSet.xml

```diff
@@ -133,10 +133,9 @@
     <required>true</required>
     <comment>What components are using this price set?</comment>
     <pseudoconstant>
-      <table>civicrm_component</table>
-      <keyColumn>id</keyColumn>
-      <labelColumn>name</labelColumn>
+      <callback>CRM_Price_BAO_PriceSet::getExtendsOptions</callback>
     </pseudoconstant>
+    <serialize>SEPARATOR_TRIMMED</serialize>
```

Comment:
Is this correct - my reading of the BAO is it should be `SERIALIZE_SEPARATOR_BOOKEND`

https://github.com/civicrm/civicrm-core/blob/5749f768184574a79bcbdaf3f788acb3b5d02575/CRM/Price/BAO/PriceSet.php#L48-L58

Reply:
Hmm, you're right... except that code falls over if you pass in an int, it skips the serialization process. And our sample data omits the seralization too. I'll see if I can fix it.